### DISCUSSION TITLE
Share ScreenShot Plugin

### DISCUSSION THREAD

**Discussion by Awws27** [runelite/runelite#17978] [Ideas]
Idea: Plugin that can share a screenshot taken by this tool with friends or clan members or channel. Toggle option. Accept/ decline option.

**Comment by raiyni** [comment]
Dink plugin

**Reply by Awws27** [reply]
Thats automated, I was thinking something manual. Just manual screenshots that you take with like a snipping tool type of deal. So take a image of something specific and share it.

**Reply by YvesW** [reply]
Dink has a custom chat message option, so you can probably configure it to send a screenshot when typing e.g. `::screenie`

**Reply by raiyni** [reply]
press the screenshot button or use sharex?
